test/unit/org/apache/cassandra/index/sai/cql/DropIndexWhileQueryingTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think we need the DS license header for new files with Apache license, instead of the ASF header.

Reply:
Ops, I will blame this one on the IDE as it added it automatically, good catch, thanks!